packages/core/src/2d/assembler/SimpleSpriteAssembler.ts

```diff
@@ -45,25 +43,46 @@ export class SimpleSpriteAssembler {
     //  |   |
     //  0 - 1
     // ---------------
-    // Update positions.
+    // Update positions
     const spritePositions = sprite._getPositions();
-    const { positions } = renderer._verticesData;
-    for (let i = 0; i < 4; i++) {
+    const { _chunk: chunk } = renderer;
+    const vertices = chunk.data.vertices;
+    for (let i = 0, o = chunk.vertexArea.start; i < 4; ++i, o += 9) {
       const { x, y } = spritePositions[i];
-      positions[i].set(wE[0] * x + wE[4] * y + wE[12], wE[1] * x + wE[5] * y + wE[13], wE[2] * x + wE[6] * y + wE[14]);
+      vertices[o] = wE[0] * x + wE[4] * y + wE[12];
+      vertices[o + 1] = wE[1] * x + wE[5] * y + wE[13];
+      vertices[o + 2] = wE[2] * x + wE[6] * y + wE[14];
```

Comment:
Can opt performance,but should opt bounds compute
```ts
static updatePositions(renderer: SpriteRenderer | SpriteMask): void {
    const { width, height, sprite } = renderer;
    const { x: pivotX, y: pivotY } = sprite.pivot;
    // Renderer's worldMatrix
    const worldMatrix = SimpleSpriteAssembler._worldMatrix;
    // Parent's worldMatrix
    const wE = renderer.entity.transform.worldMatrix.elements;
    const sX = renderer.flipX ? -width : width;
    const sY = renderer.flipY ? -height : height;
    const m0 = wE[0] * sX;
    const m1 = wE[1] * sX;
    const m2 = wE[2] * sX;
    const m4 = wE[4] * sY;
    const m5 = wE[5] * sY;
    const m6 = wE[6] * sY;
    const m12 = wE[12] - pivotX * m0 - pivotY * m4;
    const m13 = wE[13] - pivotX * m1 - pivotY * m5;
    const m14 = wE[14] - pivotX * m2 - pivotY * m6;

    // ---------------
    //  2 - 3
    //  |   |
    //  0 - 1
    // ---------------
    // Update positions
    const spritePositions = sprite._getPositions();
    const chunk = renderer._chunk;
    const vertices = chunk.data.vertices;
    for (let i = 0, o = chunk.vertexArea.start; i < 4; ++i, o += 9) {
      const { x, y } = spritePositions[i];
      vertices[o] = m0 * x + m4 * y + m12;
      vertices[o + 1] = m1 * x + m5 * y + m13;
      vertices[o + 2] = m2 * x + m6 * y + m14;
    }

    // Todo: should modify bounds compute method
    BoundingBox.transform(sprite._getBounds(), worldMatrix, renderer._bounds);
  }
```

Reply:
Will mention a new one after this PR.